sdk/batch/azure-compute-batch/src/main/java/com/azure/compute/batch/models/ReimageBatchNodeOptions.java

```diff
@@ -0,0 +1,5 @@
+package com.azure.compute.batch.models;
+
+public class ReimageBatchNodeOptions extends BatchBaseOptions {
```

Comment:
Can we find a way to separate out these manually-created classes from generated models? It seems like it'd be easiest if we maintained the ability to delete and regenerate the models folder as-needed.

Reply:
Dave and I discussed this comment and have decided to keep the manually-created classes in the **models** folder for these reasons:

1. The SDK team seems to prefer that we keep all classes in the **models** folder (we used to have `BatchSharedKeyCredentials` in a separate folder and they asked us during their review to move it to the **models** folder) 
2. This is throw-away code that we will revert once we get options bag support so the organization doesn't matter too much anyways